date.md

```diff
@@ -10,11 +10,18 @@ The `DATE` [data type](data-types.html) stores a year, month, and day.
 
 ## Format
 
-When inserting into a `DATE` column, format the value as `DATE '2016-01-25'`.
+Values of type `DATE` must be expressed using an
```

Comment:
Let's change this section title to `Syntax`.

Also, there's a count mismatch between "values" (plural) and "interpreted literal" (singular), etc. Let's rephrase slightly to:



```
Values of type `DATE` must be expressed as [interpreted literals](sql-constants.html#interpreted-literals) or string literals [annotated with](sql-expressions.html#explicitly-typed-expressions) type `DATE` or [coerced to](sql-expressions.html#explicit-type-coercions) type `DATE`.
```